src/Tizen.NUI.Components/Controls/AlertDialog.cs

```diff
@@ -133,8 +150,10 @@ public override void ApplyStyle(ViewStyle viewStyle)
 
         /// <summary>
         /// Title text of AlertDialog.
+        /// Title text is set to TitleContent's Text if TitleContent is TextLabel.
+        /// If TitleContent's Text is set manually by user, then it is not guaranteed that TitleContent's Text is the same with Title text.
```

Comment:
do you mean that if it's changed later then they might differ?

Reply:
Yes, we provide a property Text.
But we cannot prohibit user from setting TitleContent's Text manually.
Therefore, they might be differ if user sets TitleContent's Text manually.